Update ADHD dataset in examples to MAIN (#1953)
nilearn/nilearn: doc/connectivity/resting_state_networks.rst

.. _extracting_rsn:

==================================================
Extracting functional brain networks: ICA and related
==================================================

.. topic:: **Page summary**

   This page demonstrates the use of multi-subject Independent Component

>    Analysis (ICA) of naturalistic stimuli based movie watching fMRI data

Reviewer: ```suggestion
   Analysis (ICA) of movie-watching fMRI data
```

Data preparation: retrieving example data
-----------------------------------------

We will use down-sample data from the `children resting-state dataset
<https://openneuro.org/datasets/ds000228/versions/1.0.0>`_ has been

> preprocessed using `FMRIPrep <https://github.com/poldracklab/fmriprep>`_.

Reviewer: Slight preference:
```suggestion
preprocessed using `FMRIPrep <https://fmriprep.readthedocs.io>`_.
```

> We use nilearn

Reviewer: Again, I think we might want to link OSF here. It gives the exact fMRIPrep "methods section" as well as the Nilearn commands used in downsampling.

Data preparation: retrieving example data
-----------------------------------------

> We will use down-sampled data from the `adults and children watching movie dataset

Reviewer: ```suggestion
We will use down-sampled data from the `brain development dataset
```
